torch/optim/adam.py

```diff
@@ -27,10 +29,10 @@ class Adam(Optimizer):
     def __init__(
         self,
         params: ParamsT,
-        lr: Union[float, Tensor] = 1e-3,
+        lr=1e-3,
```

Comment:
looks like there are more places you accidentally removed

Reply:
fixed. sorry for this.